### PR TITLE
Adds browser environment and whatwg-fetch plugin

### PR DESCRIPTION
**Pull request by agondray**
- to allow window global variable so the linter won't think it's undefined
  - to tell the linter that fetch() exists and is not undefined after requiring
    or importing whatwg-fetch

### CODE REVIEW

File: eslint/index.js [hunk 1]
```diff
@@ -4,13 +4,15 @@ module.exports = {
...
+    "jest": true,
+    "browser": true
   },
   "parser": "babel-eslint",
   "plugins": [
     "import",
     "react",
-    "filenames"
+    "filenames",
+    "whatwg-fetch"
```
Comment (kylealwyn) [inline eslint/index.js]: Need to add this to the dependencies

**Review by kylealwyn — CHANGES_REQUESTED** [review]

File: eslint/index.js (hunk 1 above)
Comment (agondray) [reply eslint/index.js]: @kylealwyn 
Added. 👍

Comment (berfarah) [reply eslint/index.js]: Any chance we can skip that plugin? It's not supported by CodeClimate and we could add the global for `fetch` manually.

Comment (agondray) [reply eslint/index.js]: @berfarah 
I think that should be fine. We just need to make a note somewhere telling engineers to install the plugin globally.

Comment (agondray) [reply eslint/index.js]: @kylealwyn 
Talked with @berfarah . I'll remove the `whatwg-fetch` dependency and add will instead add a global rule for it in the main app.

**Review by kylealwyn — APPROVED** [review]

**Pull request merged**